cmd/influxd/main.go

```diff
@@ -29,10 +30,11 @@ func main() {
 
 	influxdb.SetBuildInfo(version, commit, date)
 
-	rootCmd := launcher.NewInfluxdCommand(context.Background())
+	v := viper.New()
+	rootCmd := launcher.NewInfluxdCommand(context.Background(), v)
 	// upgrade binds options to env variables, so it must be added after rootCmd is initialized
-	rootCmd.AddCommand(upgrade.NewCommand())
-	rootCmd.AddCommand(inspect.NewCommand())
+	rootCmd.AddCommand(upgrade.NewCommand(v))
+	rootCmd.AddCommand(inspect.NewCommand(v))
 	rootCmd.AddCommand(versionCmd())
```

Comment:
Does the version command need the viper for anything?

Reply:
It doesn't, as it just returns a struct